test/infrastructure/docker/api/v1alpha4/dockercluster_default.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha4
+
+func (s *DockerClusterSpec) setDefaults() {}
```

Comment:
Let's hear also other comments, but for consistency with the other webhooks I would like those function to be implemented in  dockercluster_webhook.go as `func defaultDockerClusterSpec(s *DockerClusterSpec) `

Reply:
+1 for consistency (same for dockercluster_validation.go I guess?)